### ISSUE TITLE
seccomp not supported on Travis CI

### ISSUE THREAD

**Issue by felquis** [oblador/loki#10]
Hi, it may help you and maybe you could help me too

I'm runnign this branch [shooting-stats](https://github.com/pagarme/pilot/tree/shooting-stars) you can see  .travis.yml in there, using travis it exits 1 with:

```txt
docker: Error response from daemon: linux seccomp: seccomp profiles are not supported on this daemon, you cannot specify a custom seccomp profile.
```

:100: :fire: :+1:

**Comment by felquis** [comment]
The same happens on Circle CI :smile:

**Comment by oblador** [comment]
Thanks for your bug report, please have a look at https://github.com/oblador/loki/pull/11 and see if that fixes your problem 😄

**Comment by oblador** [comment]
Fixed in [0.6.2](https://github.com/oblador/loki/releases/tag/v0.6.2).

**Comment by felquis** [comment]
Thank you very much! :beers:
